experimenter/experimenter/experiments/tests/api/v8/test_serializers.py

```diff
@@ -94,6 +97,9 @@ def test_expected_schema_with_desktop(self):
                 "localizations": None,
                 "locales": ["en-US"],
                 "publishedDate": experiment.published_date,
+                "isFirefoxLabsOptIn": False,
+                "firefoxLabsTitle": None,
+                "firefoxLabsDescription": None,
             },
         )
 
```

Comment:
Would also suggest to add a test for the non-default values

Reply:
hmm yep I agree